lim_test.py

```diff
@@ -0,0 +1,259 @@
+'''tests for the longintmath.py module'''
+import unittest
+import longintmath as lim
+
+class TestLIMFunc(unittest.TestCase):
+    '''tests for the long_int_(mathfunction) functions'''
+    def setUp(self):
+        self.long_big = 10**1000
+        self.float_big = 1.0e+300
+        self.float_sm = 1.0e-100
+
+    def tearDown(self):
+        del self.long_big
+        del self.float_big
+        del self.float_sm
+
+    def error_tester(self, result, expected):
```

Comment:
yup.  i tried assertAlmostEqual, but it has two problems.   1 - onlyworks floats.  overflowError for longs.  2 - it's asserting from absolute decimal.  so 3e300 -  2.999999999999e300 is 1e292 and it would only be true is the diff were <1e-7. (similarly, can't float div longs)

you mean like assertWithinError ?